manifests/client.pp

```diff
@@ -36,6 +36,7 @@
   Optional[Stdlib::Absolutepath] $tls_cacertdir = undef,
   $tls_checkpeer                                = undef,
   $tls_reqcert                                  = undef,
+  Optional[Enum['absent', 'on', 'true', 'yes', 'off', 'false', 'no']] $tls_moznss_compatibility = undef,  # lint:ignore:quoted_booleans
```

Comment:
This is getting a big long. Could you make a type in the `types` directory?

See https://github.com/camptocamp/puppet-systemd/blob/master/types/dropin.pp if you need an example

Reply:
Added.  It may not have shortened it as much as you'd like.  I broke out the "values the variable can have" into a type, and left the values of 'absent' and Undef (in place of Optional) in the manifest since I figured those weren't "types" material but more of "how does puppet handle this when it's not needed"  but I can be argued off that if you think I drew the proverbial line in the wrong place.